src/components/course/AssistantExport.js

```diff
@@ -22,7 +27,19 @@ const ExportModal = ({ showModal, handleClose, assignmentExport, assignmentTitle
     </Modal>
 );
 
-const DownloadButton = ({ assignmentExport, assignmentTitle, json = false, csv = false }) => {
+function handleClick(assignmentExport, courseId) {
+    //get request
+    //axios is promise based, we could make a fire&forget out of it? Does backend return anything?
+    axios.get('/api/admins/courses/' + courseId + '/assignments/' + assignmentExport.assignmentExport.assignmentId + '/reevaluate')
+        .then((response) => {
+            return response;
+        })
+        .catch((error) => {
+            console.log(error);
+        });
+}
```

Comment:
Could you rewrite this to use `fetch` instead of axios? I would not want to add an entire library just for this one request 😄 
Also how is this working without sending the auth token? 😅

Reply:
Could you also put it into a separate service file like the export one?

Reply:
Haha will do thanks. I had deactivated auth in local dev env this is why it wasnt on my radar 😅

Reply:
Done. Please check again if this is an acceptable solution :)